executor/aggfuncs/func_first_row.go

```diff
@@ -52,6 +52,8 @@ type basePartialResult4FirstRow struct {
 	// gotFirstRow indicates whether the first row has been got,
 	// if so, we would avoid evaluating the values of the remained rows.
 	gotFirstRow bool
+	// finishedRows indicates the number of rows has been got.
+	finishedRows int
```

Comment:
Can use `gotFirstRow` to check it?

Reply:
Maybe we can't use `gotFirstRow`. Like the example in the issue15284, when we meet the second 1, it will also return true which will stop the scan and return the wrong result.